### PR TITLE
[vault-env] added VAULT_* environment sanitization

### PR DESCRIPTION
**Pull request by dennybaa**
Hello @bonifaido,

I tried to play around with golang and have come up with this. It addresses #341 issue. Please let me know your considerations.

### CODE REVIEW

File: cmd/vault-env/main.go [hunk 1]
```diff
@@ -26,6 +26,45 @@ import (
...
+	"VAULT_MFA",
+	"VAULT_ROLE",
+	"VAULT_PATH",
+}
+
+// Appends variable an entry (name=value) into the environ list.
+// VAULT_* variables are not populated into this list.
+func (environ *sanitizedEnviron) append(name interface{}, value interface{}) {
+	name, value := name.(string), value.(string)
+	if sanitizeEnvmap == nil {
```
Comment (bonifaido) [inline cmd/vault-env/main.go]: I think you can fill this map in var declaration, so this list and the loop is not needed.

Comment (dennybaa) [reply cmd/vault-env/main.go]: actually might, but I don't know how... btw fixed the compilation issues I believe.

Comment (dennybaa) [reply cmd/vault-env/main.go]: you mean?
```go
var keys = map[string]bool{
  "VAULT_x": true,
}
```

Comment (bonifaido) [reply cmd/vault-env/main.go]: Exactly!

**Pull request merged**